lib/http-client.js

```diff
@@ -56,6 +68,17 @@ export default class PodiumHttpClient {
             connections,
             pipelining, // TODO unknown option, consider removing
         });
+
+        if (fallback) {
+            this.fallback(fallback);
+        }
+
+        this.#breaker.on('open', () => {
+            this.emit('open');
+        });
```

Comment:
Are we using the open and closed terminology correctly? It's confusing since open means "tripped". I keep mucking this up in my head.

Reply:
it's what the library uses 🤷  but I think that we could ignore these in the initial version and then we can see wether it's actually needed at all to have this level of control.